src/core/Icon/Icon.tsx

```diff
@@ -38,7 +38,7 @@ export const getSvgSize = (size: IconSize) => {
     case "3xl":
       return 48;
     case "4xl":
-      return 150;
+      return 100;
     case "5xl":
       return 180;
```

Comment:
is the 5xl size being used or we can also reduce this if we have a future use case for it @S-ayanide

Reply:
We might need the 5xl size for use cases where the icons are attached with the stepper and in the analytics icons, it'll be trial and error mostly